plugins/modules/dcnm_network.py

```diff
@@ -181,11 +181,19 @@
           deploy:
             description:
             - Per switch knob to control whether to deploy the attachment
+            - This knob has been deprecated from release 2.1.0 onwards and will not have any functional
```

Comment:
Please specify this as `Ansible NDFC Collection Version 2.1.0`

Reply:
Changes done as suggested